### PR TITLE
Update PSP samples.

### PR DESCRIPTION
**Pull request by myidpt**
- Remove the PSP config for node agent.
- Add `- NET_RAW` capability for the PSP config of all workloads.
- Remove `allowedHostPaths` from the PSP config of all workloads.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by rlenglet** [comment]
This needs to be cherrypicked, right?

**Comment by myidpt** [comment]
Correct. Auto-cherry-pick to 1.5. I will manually cherry-pick to 1.4 as well (which won't remove the node-agent PSP).

File: samples/security/psp/sidecar-psp.yaml [hunk 1]
```diff
@@ -3,15 +3,12 @@
 apiVersion: extensions/v1beta1
 kind: PodSecurityPolicy
 metadata:
-  name: istio-sds-uds
```
Comment (incfly) [inline samples/security/psp/sidecar-psp.yaml]: fyi, above code link is gone as part of https://github.com/istio/istio.io/pull/6644
currently seems no instructions reference how to use sample files on istio.io

Comment (incfly) [reply samples/security/psp/sidecar-psp.yaml]: https://istio.io/docs/tasks/security/auth-sds/#increasing-security-with-pod-security-policies

**Pull request merged**